docs/src/main/sphinx/release/release-437.md

```diff
@@ -0,0 +1,36 @@
+# Release 437 (24 Jan 2024)
+
+## General
+
+* Add support for `char(n)` values in {func}`to_utf8`. ({issue}`20158`)
+* Add support for `char(n)` values in {func}`lpad`. ({issue}`16907`)
+* {{breaking}} Replace the `exchange.compression-enabled` configuration property
+  and `exchange_compression` session property with
+  [the`exchange.compression-codec`and `exchange_compression_codec` properties](prop-exchange-compression-codec),
+  respectively. ({issue}`20274`)
+* {{breaking}} Replace the `spill-compression-enabled` configuration property 
+  with [the `spill-compression-codec` property](prop-spill-compression-codec). ({issue}`20274`)
+* {{breaking}} Remove the deprecated `experimental.spill-compression-enabled`
+  configuration property. ({issue}`20274`)
+* Fix permanently stale materialized views when using pass-through table
```

Comment:
There's no such thing as a "pass-through" table function. Also, this issue has nothing to do with whether it's a a function that implements query pass-through or not -- all that matters is that the view references a table function.